.circleci/config.yml

```diff
@@ -158,6 +167,7 @@ jobs:
     steps:
       - checkout
       - <<: *install_suitesparse
+      - <<: *install_functorch
```

Comment:
Need to add functorch to the gpu tests also. For the cpu test we are using `nox` which runs `pip install -e .`, so we also need to add `functorch` to the main requirements file.

Reply:
Now it works for CircleCI.